javatests/arcs/core/storage/driver/DatabaseDriverProviderTest.kt

```diff
@@ -122,6 +159,86 @@ class DatabaseDriverProviderTest {
     assertThat(singletonDriver.storageKey).isEqualTo(key)
   }
 
+  @Test
+  fun removeAllEntities() = runBlockingTest {
+    val mockManager = mock<DatabaseManager>()
+    provider.configure(mockManager, schemaHashLookup::get)
+
+    provider.removeAllEntities()
+
+    verify(mockManager).removeAllEntities()
```

Comment:
this is a suspend function as well, does the verify work?

Reply:
Discussed in chat: Yes, this verify will work. This has been... verified with a manual test. Pasting the reasoning below, for posterity: 

> I think verify is ok here because the problem is with matchers on suspend functions: They don't know how to do the match given all the lambdas that are created.